File: SDKs/Aspose.Words_Cloud_SDK_for_CSharp/src/Com/Aspose/Words/Model/ReplaceTextResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class ReplaceTextResponse {
    public int Matches { get; set; }

    public Link DocumentLink { get; set; }

    public string Code { get; set; }

    public string Status { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ReplaceTextResponse {\n");
      sb.Append("  Matches: ").Append(Matches).Append("\n");
      sb.Append("  DocumentLink: ").Append(DocumentLink).Append("\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  Status: ").Append(Status).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: Examples/Android/Aspose-Words-Cloud-SDK-for-Android/src/main/java/com/aspose/words/model/ReplaceTextResponse.java
package com.aspose.words.model;

import com.aspose.words.model.Link;
public class ReplaceTextResponse {
  private Integer Matches = null;
  private Link DocumentLink = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getMatches
	 * Gets Integer
	 * @return Matches
	 */
  public Integer getMatches() {
    return Matches;
  }

	/**
	 * setMatches
	 * Sets Integer
	 * @param Matches Integer
	 */
  public void setMatches(Integer Matches) {
    this.Matches = Matches;
  }

  /**
	 * getDocumentLink
	 * Gets Link
	 * @return DocumentLink
	 */
  public Link getDocumentLink() {
    return DocumentLink;
  }

	/**
	 * setDocumentLink
	 * Sets Link
	 * @param DocumentLink Link
	 */
  public void setDocumentLink(Link DocumentLink) {
    this.DocumentLink = DocumentLink;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ReplaceTextResponse {\n");
    sb.append("  Matches: ").append(Matches).append("\n");
    sb.append("  DocumentLink: ").append(DocumentLink).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

